autoPyTorch/api/base_task.py

```diff
@@ -888,6 +909,11 @@ def _search(
                 pd.DataFrame(self.ensemble_performance_history).to_json(
                     os.path.join(self._backend.internals_directory, 'ensemble_history.json'))
 
+        # Add run history of traditional algorithms to the full run history
+        if hasattr(self, 'run_history') and self.run_history is not None and traditional_run_history is not None:
```

Comment:
`None not in [self.run_history, traditional_run_history]` is a bit shorter:-)

Reply:
Actually according to @franchuterivera requested changes, this line is no longer needed